api/src/utils/query-actions.js

```diff
@@ -28,8 +28,8 @@ exports.publish = async function(obj) {
       obj.datePublished = date;
       obj.markModified('datePublished');
 
-      obj.dateUpdated = date;
-      obj.markModified('dateUpdated');
+      obj.publishedBy = auth_payload && auth_payload.displayName;
```

Comment:
I don't think we want display name here.  We should be using a unique identifier here for the user: idir_userid.  We may want to start storing a complex object here instead of a string.  ie: the preferred_username & idir_userid.  I don't think we should store displayName at all.

Reply:
TBH Im not sure what values come in the auth_payload.  We seem to use `displayName` in the vast majority of places, so I just followed suit.  But, can change it to whatever is most appropriate of course.